tests/types/test_hstore.py

```diff
@@ -1,8 +1,14 @@
 import pytest
 
 import psycopg
+from psycopg.pq import Format
 from psycopg.types import TypeInfo
-from psycopg.types.hstore import HstoreLoader, register_hstore
+from psycopg.types.hstore import (
+    HstoreLoader,
+    _make_hstore_binary_dumper,
+    _make_hstore_binary_loader,
+    register_hstore,
+)
```

Comment:
I hate this import style. If this is still necessary after the encoding changes:
```suggestion
from psycopg.types.hstore import HstoreLoader, _make_hstore_binary_dumper
from psycopg.types.hstore import _make_hstore_binary_loader, register_hstore
```